.github/workflows/weekly_flatpak.yml

```diff
@@ -12,11 +12,11 @@ jobs:
     # again
     runs-on: ubuntu-24.04
     container:
-      image: bilelmoussaoui/flatpak-github-actions:gnome-nightly
+      image: bilelmoussaoui/flatpak-github-actions:gnome-46
```

Comment:
This isn't strictly necessary. We could use nightly. I just figured this would be more stable, and it's okay if we lag behind a bit after new GNOME releases.

Reply:
True, we can update every now and then